partner_multi_relation_tabs/models/res_partner.py

```diff
@@ -15,11 +15,12 @@ class ResPartner(models.Model):
 
     @api.multi
     def browse(self, arg=None, prefetch=None):
-        for tab in self._get_tabs():
-            fieldname = tab.get_fieldname()
-            if fieldname not in self._fields:
-                # Check this for performance reasons.
-                self.add_field(tab)
+        if "show_relation_tab" in self.env.context:
```

Comment:
I think the context key should perhaps have a name like 'update_relation_tab'. The browse record could also be used programmatically to get at the partners that are related, in a way that places them on a tab. For instance suppose you have a "board tab" that lists chairperson, secretary and member relations for an organisation. You might use the partners on the board tab to send all of them an email.

Reply:
Changed context name it to `update_relation_tab`